tests/test_hermes_doctor_bank_routing.py

```
@@ -0,0 +1,463 @@
+        assert not named_db.exists()
+
+
+    def test_sleep_routing_unaffected_dry_run(self, tmp_path, monkeypatch):
+        """Sleep (dry-run) still routes to the resolved named bank beam.
+
+        `doctor` guard only fires for cmd == "doctor"; sleep keeps its
+        existing routing. We assert the resolved bank beam is used and no
+        new rejection/failure path is introduced for sleep.
+        """
+        home = tmp_path / "profiles" / "work"
+        _write_config(home, True)
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)